src/vs/workbench/browser/parts/statusbar/statusbarPart.ts

```diff
@@ -534,7 +534,14 @@ export class StatusbarPart extends Part implements IStatusbarService {
 
 				/* Status bar focus outline */
 				.monaco-workbench .part.statusbar:focus {
-					outline-color: ${statusBarFocusColor};
+					outline: 0;
+					border: 1px solid ${statusBarFocusColor};
+					-webkit-border-bottom-right-radius: 10px;
+					-webkit-border-bottom-left-radius: 10px;
+					-moz-border-radius-bottomright: 10px;
+					-moz-border-radius-bottomleft: 10px;
+					border-bottom-right-radius: 10px;
+					border-bottom-left-radius: 10px;
```

Comment:
In many situations we use `outline` to avoid layout jank when an element doesn't always feature a border. Many themes (like the default dark theme) don't use a border on the status bar unless focused, so the elements might shift around when adding and removing it on focus. Have you verified that moving to a border doesn't cause this?